### PR TITLE
Fix purview scanning URL

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

**Comment by jianghaolu** [comment]
@qwordy FYI

File: sdk/purview/azure-analytics-purview-scanning/README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Set the values of the client ID, tenant ID, and client secret of the AAD applica
 <!-- embedme ./src/samples/java/com/azure/analytics/purview/scanning/ReadmeSamples.java#L22-L25 -->
 ```java
 SystemScanRulesetsBaseClient client = new PurviewScanningClientBuilder()
-    .endpoint(System.getenv("<account-name>.scanning.purview.azure.com"))
+    .endpoint(System.getenv("<account-name>.scan.purview.azure.com"))
```
Comment (qwordy) [inline sdk/purview/azure-analytics-purview-scanning/README.md:63]: `getenv` accepts a name of the environment variable. Is it a valid variable name?

Comment (jianghaolu) [reply sdk/purview/azure-analytics-purview-scanning/README.md:63]: No - let me remove the System.getenv call

Comment (jianghaolu) [reply sdk/purview/azure-analytics-purview-scanning/README.md]: Fixed.

**Comment by jianghaolu** [comment]
Overriding check enforcer since this only touches readme.

**Comment by jianghaolu** [comment]
@qwordy @srnagar Feel free to approve & merge.

**Review by srnagar — APPROVED** [review]

**Pull request merged**